native/machine-coding/Finance Tracker/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Finance Tracker</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="../../helpers/navbar.js" type="module"></script>
```

Comment:
add header.js only to the head (as per the new standards)
refer [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/javascript/machine-coding/counter/index.html)